src/lsp/cobol_lsp/package.toml

```
@@ -62,7 +62,7 @@ cobol_typeck = "version"
 superbol_preprocs = "version"
 superbol_project = "version"
 jsonrpc = ">=1.15"
-lsp = ">=1.15 <1.16"
+lsp = "=1.18.0"
```

```suggestion
lsp = "1.18"
```
(This implicitly means "≥1.18"; should be enough)